app/services/validation_beta_service.rb

```diff
@@ -22,6 +22,14 @@ def invite_schools(array_of_urns:, participant_research:, coordinator_research:)
     end
   end
 
+  def notify_induction_coordinators_about_validation
+    School.eligible.find_each do |school|
+      if added_participants_and_not_in_beta(school) || chosen_programme_but_not_added_participants(school)
```

Comment:
This condition simplifies to "chosen programme and not in beta"

Reply:
> 2. SITs who have signed in AND chosen their programme, but have not added their ECTs and mentors

I understood this as the second set of schools could be in beta, will clarify

Reply:
Got you, if they haven't added participants they wouldn't be invited to beta